### PR TITLE
Allow db_path to be set to ":memory:"

### PR DESCRIPTION
**Pull request by cmd-ntrf**
Fix issue #69

### CODE REVIEW

**Comment by Zsailer** [comment]
This is great, thank you @cmd-ntrf. Ideally, we have a unit test to cover this config option to make sure we don't accidentally remove it in the future. 

Would you like to create such a unit test? Otherwise, I can take over this PR, write such a test, and merge.

File: tests/test_manager.py [hunk 1]
```diff
@@ -148,6 +148,13 @@ def test_validates_db_path(jp_root_dir, any_fid_manager_class):
         )
 
 
+def test_in_memory_db(jp_root_dir, any_fid_manager_class):
+    fid_manager = any_fid_manager_class(root_dir=str(jp_root_dir), db_path=":memory:")
+    database_list = fid_manager.con.execute("PRAGMA database_list")
+    for id_, name, path in database_list:
+        assert path == ""
```
[the remark this answers is not in the document]
Comment (Zsailer) [reply tests/test_manager.py:155]: I think the unit test needed here is to just check that `":memory:"` doesn't ever throw a traitlets error during type validation, i.e. it is always an acceptable value. No need to actually test the database connection—we can assume sqlite covers this in their unit tests. 

The purpose of the unit test is catch/prevent us from removing `":memory:"` as an acceptable value by accident in the future.

[the remark this answers is not in the document]
Comment (dlqqq) [reply tests/test_manager.py:155]: > I think the unit test needed here is to just check that ":memory:" doesn't ever throw a traitlets error during type validation, i.e. it is always an acceptable value. No need to actually test the database connection—we can assume sqlite covers this in their unit tests.

Yes, I agree. That is the point I'm making with this suggestion.

**Review by Zsailer — APPROVED** [review]

**Comment by Zsailer** [comment]
This change released today: https://github.com/jupyter-server/jupyter_server_fileid/releases/tag/v0.9.1

Thanks @cmd-ntrf!

**Comment by dlqqq** [comment]
@Zsailer Thanks for merging & releasing this PR while I was away for vacation! 🎉

**Pull request merged**